boto3/ec2/deletetags.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from boto3.resources.model import Action
+from boto3.docs.docstring import ActionDocstring
+from boto3.utils import inject_attribute
+
+
+def create_delete_tags_action(event_emitter):
+    action_model = {
+        'request': {
+            'operation': 'DeleteTags',
+            'params': [{
+                'target': 'Resources[0]',
+                'source': 'identifier',
+                'name': 'Id'
+            }]
+        }
+    }
+    return CustomModeledAction('delete_tags', action_model, delete_tags, event_emitter)
+
+
+def delete_tags(self, **kwargs):
+    kwargs['Resources'] = [self.id]
+    return self.meta.client.delete_tags(**kwargs)
+
+
+class CustomModeledAction(object):
```

Comment:
I would like to put this in `utils.py`, but importing `ActionDocstring` causes a circular dependency as it depends on `ServiceContext`. To get it moved, I could:

1. Move `ServiceContext` to its own file, or somewhere more appropriate
2. Import `ActionDocstring` after `ServiceContext` is defined, or
3. Lazily import `ActionDocstring`  within `inject`

Thoughts?